dist/gcc-compatible/EverCrypt_Hash.c

```diff
@@ -1249,7 +1242,7 @@ EverCrypt_Hash_Incremental_init(Hacl_Streaming_Functor_state_s___EverCrypt_Hash_
   Hacl_Streaming_Functor_state_s___EverCrypt_Hash_state_s____ scrut = *s;
   uint8_t *buf = scrut.buf;
   EverCrypt_Hash_state_s *block_state = scrut.block_state;
-  Spec_Hash_Definitions_hash_alg i = EverCrypt_Hash_alg_of_state(block_state);
+  (void)EverCrypt_Hash_alg_of_state(block_state);
```

Comment:
this is the worst: krml doesn't have an inter-procedural analysis so doesn't know that this function is "pure" (for what definition of pure?) and therefore "voids" away the result, but this is definitely strange -- need to work on source code to get rid of this